### ISSUE TITLE
.select()

### ISSUE THREAD

**Issue by mikermcneil** [balderdashy/waterline#73]
Support for both sorts of usages:

Traditional `SELECT`:
````javascript
User.find().select('name', 'age').exec(cb);
User.find().select(['name', 'age']).exec(cb);
````

`Prune` is an exclude (inverse of select)
````javascript
User.find().prune('password', 'apiKey').exec(cb);
User.find().prune(['password', 'id']).exec(cb);
````

**Comment by FWoelffel** [comment]
+1 Would love it

**Comment by tuoxie007** [comment]
+1

**Comment by dmarcelino** [comment]
For those that may not be aware waterline supports `select` in the form of:
``` javascript
// Returns only the field name
Model.find({ where: { age: { '<': 30 }, select: ['name'] }})
```

Docs: https://github.com/balderdashy/waterline-docs/blob/master/query-language.md#select

**Comment by tuoxie007** [comment]
cool!

发自我的 iPhone

**Comment by iwhitfield** [comment]
For consistency, it would make sense if this had a dedicated function, as every other option does. 

FWIW I think the initial proposal from @mikermcneil is the best way to go, it seems natural.

**Comment by sailsbot** [comment]
Thanks for posting, @mikermcneil.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by vishnumishra** [comment]
Can we have the method 

> User.find().select({'name':{'AS':'userName'},{id:{AS:'UserId'}})
